### PR TITLE
Add feature viewer to the different sections

### PR DESCRIPTION
**Pull request by xwatkins**
- [ ] What is the link to the Jira that this PR is for?
Adding the feature viewer to the relevant sections as described here http://ebi-uniprot.github.io/ProtVista/userGuide.html#feature-categories-and-types
- [ ] Have you written tests and do these have >= 75% coverage of the code that you are contributing?
Just needed snapshot update
- [ ] Is it possible to visually evaluate this PR in the browser? If so, what are the steps to do this (eg specific URLs or sections)?
Go to entry view, check for presence of feature viewer/feature table
- [ ] Are there any linting errors that required you to add exceptions?
- [ ] If there are backend requests, have you checked for the existence of the resource's attributes before attempting access?

### CODE REVIEW

File: src/model/FeaturesView.tsx [hunk 1]
```diff
@@ -63,13 +66,18 @@ const columns = {
   },
   description: {
     label: 'Description',
-    resolver: (d: ProtvistaFeature) => d.description,
+    resolver: (d: ProtvistaFeature) => {
+      return `${d.description} ${d.evidences &&
```
Comment (dlrice) [inline src/model/FeaturesView.tsx]: `(x) => {return y}` blocks can become `x => y`

File: src/model/FeaturesView.tsx [hunk 2]
```diff
@@ -63,13 +66,18 @@ const columns = {
...
+        d.evidences.map(evidence => `(${evidence.evidenceCode}) `)}`;
```
Comment (dlrice) [inline src/model/FeaturesView.tsx]: Embedding too much inside of template strings makes it difficult read.

File: src/model/FeaturesView.tsx [hunk 3]
```diff
@@ -63,13 +66,18 @@ const columns = {
...
+    },
   },
 };
 
 const processData = (data: Feature[], types: string[] = []) =>
   data
-    .filter(feature => types.length <= 0 || types.includes(feature.type))
+    .filter(feature => {
+      return types.length <= 0 || types.includes(feature.type);
```
Comment (dlrice) [inline src/model/FeaturesView.tsx]: `(x) => {return y}` blocks can become `x => y`

File: src/model/FeaturesView.tsx (hunk 2 above)
Comment (xwatkins) [reply src/model/FeaturesView.tsx]: This is temporary as we need a strategy for evidence tags to work both in website and in Nightingale, so using it as a placeholder.

**Pull request merged**